<?xml version="1.0"?>
<launch>
	
	<arg name="robot_model" default="mbzirc2020"/>

	<node name="estimation_node" pkg="object_estimation" type="estimation_node" respawn="false" output="screen" launch-prefix="gdb -ex run --args">

	<param name="frequency" value="1.0"/>
	<param name="robot_ns" value="$(arg robot_model)"/>
	
	<rosparam param="object_types">["pile","wall"]</rosparam>
	<!-- <rosparam param="object_types">["fire","passage"]</rosparam> -->
	<rosparam param="uav_ids">["1", "2", "3"]</rosparam>
	<param name="a_priori_info" value="true" />
	<param name="conf_file" value="$(find object_estimation)/config/conf_ch2.yaml" />

	<param name="lost_time_th" value="20.0" />      <!-- in seconds -->
	<rosparam param="association_th">[6.0,6.0]</rosparam>
	<param name="min_update_count" value="5" />
	<param name="delay_max" value="2.0" />          <!-- in seconds -->
	        
	</node>

</launch>
